tsify-next-macros/src/attrs.rs

```diff
@@ -14,8 +12,6 @@ pub struct TsifyContainerAttrs {
     pub namespace: bool,
     /// Information about how the type should be serialized.
     pub ty_config: TypeGenerationConfig,
-    /// Comments associated with the type. These will be written out to the generated Typescript.
-    pub comments: Vec<String>,
```

Comment:
Were these comments not being used? Why was this field removed?

Reply:
Correct, the static code analysis (clipping) reported that this field was never read. That was also one of the reasons the CI failed.